### PR TITLE
Add UNA to `HISTORY.rst` and merge in latest changes from `release/3.13.0`

### PR DESCRIPTION
**Pull request by phargogh**
This PR does 2 things:
* Adds UNA to `HISTORY.rst`
* Merges in the latest changes from `release/3.13.0`.

All UNA issues for the 3.13.0 release are now complete, so once this PR is merged, I will create a new PR to merge `feature/urban-nature-access` in to `release/3.13.0` and mark the model issue #722 as complete.

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -43,9 +44,63 @@ Unreleased Changes
...
+      by some users on Windows. https://github.com/natcap/invest/issues/1189
+* SDR
+    * The ``ws_id`` field is no longer a required field in the watershed vector.
+      https://github.com/natcap/invest/issues/1201
+* Seasonal Water Yield
+    * If a soil group raster contains any pixels that are not in the set of
+      allowed soil groups (anything other than 1, 2, 3 or 4), a human readable
+      exception will now be raised. https://github.com/natcap/invest/issues/1193
+* Urban Nature Access
+    * Added this new model.
```
Comment (dcdenu4) [inline HISTORY.rst:99]:
```suggestion
    * Added the Urban Nature Access model to InVEST. The model for urban nature access provides a measure of both the supply of urban nature and the demand for nature by the urban population , ultimately calculating the balance between supply and demand. See the corresponding User's Guide chapter for a complete overview.
```

Comment (dcdenu4) [reply HISTORY.rst:99]: Ugh, I couldn't format this well via a suggestion.

**Review by dcdenu4 — CHANGES_REQUESTED** [review]
Quick History suggestion and waiting for tests to pass

File: HISTORY.rst (hunk 1 above)
Comment (phargogh) [reply HISTORY.rst]: @dcdenu4 I just updated the description very much line with what you wrote in 817183a6e.  Much easier to format in an editor!

**Comment by phargogh** [comment]
@dcdenu4 I forgot about the `MODEL_SPEC` outputs!  I'll implement those and then get back to you.  This PR will probably need to wait until Monday.

**Review by dcdenu4 — CHANGES_REQUESTED** [review]
Thanks for the History update @phargogh . I'm just requesting changes via Model Spec stuff so I get pinged when you "re-request" my review.

File: src/natcap/invest/urban_nature_access.py [hunk 2]
```diff
@@ -297,6 +297,287 @@
...
+                    "bands": {1: {'type': 'number', 'units': u.count}},
+                },
+                "oversupplied_population.tif": {
+                    "about": gettext(
+                        "The population experiencing an urban nature surplus."
+                    ),
+                    "bands": {1: {'type': 'number', 'units': u.count}},
+                },
+
+
```
Comment (dcdenu4) [inline src/natcap/invest/urban_nature_access.py:449]:
```suggestion
```

Comment (dcdenu4) [reply src/natcap/invest/urban_nature_access.py:449]: Extra lines, those that suggestion only deletes one line...

File: src/natcap/invest/urban_nature_access.py [hunk 3]
```diff
@@ -297,6 +297,287 @@
...
+                "urban_nature_supply_lucode_[LUCODE].tif": {
+                    "about": gettext(
+                        "The urban nature supplied to populations due to "
+                        "the land use land cover class LUCODE."),
+                    "bands": {1: {"type": "number", "units": u.m**2/u.person}},
+                    "created_if":
+                        f"search_radius_mode == '{RADIUS_OPT_URBAN_NATURE}'",
+                },
+
+
```
Comment (dcdenu4) [inline src/natcap/invest/urban_nature_access.py:516]:
```suggestion
```

File: src/natcap/invest/urban_nature_access.py (hunk 2 above)
Comment (dcdenu4) [inline src/natcap/invest/urban_nature_access.py:449]:
```suggestion
```

**Review by dcdenu4 — CHANGES_REQUESTED** [review]
Thanks @phargogh ! I just noticed some extra lines otherwise happy to merge this in!

**Pull request merged**